### ISSUE TITLE
Create a visitor to split insertion of multiple articles

### ISSUE THREAD

**Issue by Seb35** [promethe42/duralex#10]
For instance there are 3 articles and 1 title inserted together in http://www.assemblee-nationale.fr/15/textes/0911.asp#D_Article_14.

A visitor should be created to detect such cases and split them into multiple article-definition nodes. In the example given, the name of title XI is also modified and a title-definition should also be created.

**Comment by JMLX42** [comment]
AFAIK that's already the case. There is a visitor that will split/fork edit nodes with multiple references/definitions.

**Comment by promethe42** [comment]
AFAIK that's already the case. There is a visitor that will split/fork edit nodes with multiple references/definitions.

**Comment by Seb35** [comment]
Yes, ForkEditVisitor (which btw is not used) but it operates on existing ref+def nodes. Here the case is a single quote in a single def node. The text content should be splitted, and probably ForkEditVisitor will be useful then.
